src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
+    return !(
+      document.fullscreenEnabled === false ||
+      document.webkitFullscreenEnabled === false ||
+      document.mozFullScreenEnabled === false ||
+      document.msFullscreenEnabled === false
+    )
```

Comment:
Why don't you instead of this do:
```javascript
return !!(document.fullscreenEnabled ||
  document.webkitFullscreenEnabled ||
  document.mozFullScreenEnabled ||
  document.msFullscreenEnabled
)
```
I think it reads better, and avoids comparing to `false`.